src/main/plugins/control-panel-plugin/control-panel-plugin.ts

```diff
@@ -0,0 +1,63 @@
+import { SearchResultItem } from "../../../common/search-result-item";
+import { PluginType } from "../../plugin-type";
+import { IconType } from "../../../common/icon/icon-type";
+import { TranslationSet } from "../../../common/translation/translation-set";
+import { UserConfigOptions } from "../../../common/config/user-config-options";
+import { SearchPlugin } from "../../search-plugin";
+import { ControlPanelOptions } from "../../../common/config/control-panel-options";
+import { ControlPanelItem } from "./control-panel-item";
+import { defaultControlPanelIcon } from "../../../common/icon/default-icons";
+import { ControlPanelItemsRetriever } from "./control-panel-items-retriever";
+import { Logger } from "../../../common/logger/logger";
+import * as Powershell from "node-powershell";
+
+export class ControlPanelPlugin implements SearchPlugin {
+    public pluginType = PluginType.ControlPanel;
+    private controlPanelItems: ControlPanelItem[] = [];
+
+    constructor(private config: ControlPanelOptions, private readonly logger: Logger) {}
```

Comment:
Can you please use the same pattern as the other plugins? Declare class properties before the constructor and then assign the parameters in the constructor. I suggest this:

``` typescript
private controlPanelItems: ControlPanelItem[];
private config: ControlPanelOptions;
private readonly logger: Logger;

constructor(config: ControlPanelOptions, logger: Logger) {
    this.config = config;
    this.logger = logger;
    this.controlPanelItems = [];
}
```

I know that your solution looks cleaner and is less lines of code but I want the code of ueli to be/feel consistent.